buildbot/master/files/config/factories.py

```diff
@@ -90,6 +90,7 @@ def setDefaultWorkdir(self, workdir):
     @defer.inlineCallbacks
     def run(self):
         self.is_windows = re.match('windows.*', self.builder_name) is not None
+        self.is_win_gnu = re.match('windows.*gnu', builder_name) is not None
```

Comment:
Also, you can remove the `is not None` for both of these, which will let them fit under the line limit.